### ISSUE TITLE
Implement Dijkstra’s shortest path algorithm

### ISSUE THREAD

**Issue by JASKAMAL22** [ayan-biswas0412/AlgoDocs#111]
**Please describe the proposed algorithm you want to add to this repo.**
Dijkstra’s shortest path algorithm

**Describe sources (if you want to follow any)**
A clear and concise code with relevant comments.

**Describe the languages in which you want to implement the algorithm**
Python

**Are you working on this? (Yes/No)**
Yes

**Comment by ayan-biswas0412** [comment]
> Hello @ayan-biswas0412, Can I work on it using python?

Yeah sure go ahead, your contribution will help this project to grow more.
